package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "plaid",
-  "version": "4.0.0",
+  "version": "3.2.0",
```

Comment:
my mistake before – looks like @michaelckelly released 4.0 15 days ago – so this would be `4.1.0`